ios/engine/build.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*/*}/../resources/build/build-utils.sh"
```

Comment:
```suggestion
. "${THIS_SCRIPT%/*}/../../resources/build/build-utils.sh"
```

(we normally use that pattern to indicate parent folder, so just for consistency :grin:)

Reply:
... we do?

I'll admit that I was wondering why we didn't just `/*` all the prefix-folders off, but I had no idea this was an established pattern.  Also, I'm still not clear on what said pattern actually even is... I've probably done it wrong in some of the web scripts.

Reply:
The pattern is actually `/*` to strip off build.sh, and then `../` for parent folder. So it's logically two separate operations: a string operation for `pathname(THIS_SCRIPT)` and then marking a relative path. It's all swings and roundabouts, but I feel like using `../` for parent folder is slightly clearer.